AsyncDisplayKit/ASTableView.mm

```diff
@@ -876,6 +876,35 @@ - (ASCellNode *)dataController:(ASDataController *)dataController nodeAtIndexPat
   return node;
 }
 
+- (ASCellNodeBlock)dataController:(ASDataController *)dataController nodeBlockAtIndexPath:(NSIndexPath *)indexPath {
+  if (![_asyncDataSource respondsToSelector:@selector(tableView:nodeBlockForRowAtIndexPath:)]) {
+    ASCellNode *node = [_asyncDataSource tableView:self nodeForRowAtIndexPath:indexPath];
+    ASDisplayNodeAssert([node isKindOfClass:ASCellNode.class], @"invalid node class, expected ASCellNode");
+    __weak __typeof__(self) weakSelf = self;
```

Comment:
I can't immediately think of how to deduplicate this from below, but we have 4 copies of largely-the-same code here...I wonder if we can share this as "set up cell node for use by range controller with layoutDelegate" in a different file, used by both table and collection?  Not needed for this diff, just a highly desirable to keep the file size down in the future.